keyboards/handwired/polykybd/lang/_gen_script_cols.py

```
@@ -0,0 +1,202 @@
+        elif isinstance(v,(int,float)): cells.append(["num",int(v)])
+        else: cells.append(["str",str(v)])
+    thcol[r]=cells
+cols_out["th-TH"]=thcol
+th_seq=[]
+for j,mark in enumerate(sorted(th_marks)):
+    pua=0xE180+j; add_named(f"THAI_DC_{mark:04X}", pua); th_seq.append(f"25CC {mark:04X}")
+if th_seq:
+    fonts_yaml.append({"lang":"th-TH","key":"th","block":(0xE00,0xE7F),
+                       "pua_base":0xE180,"seq":", ".join(th_seq)})
+
+json.dump(cols_out, open("/tmp/script_cols.json","w"), indent=1)
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/script_cols.json"

(S108)